build/ansible/roles/pmm-images/tasks/main.yml

```diff
@@ -144,3 +143,30 @@
     state: touch
     owner: pmm
     group: pmm
+
+# Launch pmm-managed-init first as it may update supervisord configuration on start
+- name: Generate new supervisor config
+  command: pmm-managed-init
+  register: managed_init_result
+  changed_when: True
+
+- name: Disable pmm-update-perform-init
+  ini_file:
+    path: /etc/supervisord.d/pmm.ini
+    section: program:pmm-update-perform-init
+    option: autostart
+    value: "false"
+
+# During build time, this will be the first start of supervisord.
+- name: Start supervisord
+  shell: supervisord -c /etc/supervisord.conf &
+
+- name: Run initialization playbook
+  include_role:
+    name: initialization  
+
+- name: Check supervisord logs
+  shell: sleep 10 && tail -n 200 /srv/logs/supervisord.log
+
+- name: Check grafana logs
+  shell: cat /srv/logs/grafana.log
```

Comment:
do we still need it?

Reply:
I meant whole block added

Reply:
Sorry, I'm not sure I understand the question.

Reply:
Do we really need to start supervisor and initialization playbook?

Reply:
it copies into /srv directory which we delete anyway on post-build-actions and can be done on user side.

Reply:
It's false, we don't delete the whole `/srv` directory in the `post-build` action.

Reply:
This is actually what makes the container start very fast, otherwise it would be slower.